packages/ia-topnav/src/signed-out-dropdown.js

```diff
@@ -5,6 +5,12 @@ class SignedOutDropdown extends DropdownMenu {
   static get styles() {
     return [DropdownMenu.styles, signedOutDropdownStyles];
   }
+
+  get menuClass() {
+    let menuClass = super.menuClass;
+    menuClass += this.config.hideSearch ? ' search-hidden' : '';
+    return menuClass;
```

Comment:
Question: would this be more clean?
```
    return super.menuClass + this.config.hideSearch ? ' search-hidden' : '';
```

Reply:
Yeah, it could be shortened to something like `return \`${super.menuClass}${this.config.hideSearch ? ' search-hidden' : ''}\`;` but it felt like it would be easier to quickly understand what was being returned when written in multiple statements.